src/main/webapp/frame/framec.jsp

```diff
@@ -370,16 +379,17 @@
                     </block>
                 </value>
             </block>
+            <block type="scribbler_exit_loop"></block>
             <block type="controls_if"></block>
             <block type="scribbler_wait">
                 <value name="WAITTIME">
-                    <block type="spin_integer">
-                        <field name="INT_VALUE">500</field>
+                    <block type="math_number">
+                        <field name="NUM">500</field>
                     </block>
                 </value>
-                <field name="TIMESCALE">1000</field>
```

Comment:
Was changing the time scale from 1000 to 1 a deliberate decision? I'm hoping that this was brought about as part of moving to the new C library for Spin.